test/cmd/authentication.sh

```diff
@@ -41,7 +41,6 @@ useruid="$(oc get user/scoped-user -o jsonpath='{.metadata.uid}')"
 os::cmd::expect_success_and_text "oc policy can-i --list -n '${project}' --as=scoped-user" 'get.*pods'
 os::cmd::expect_success "oc policy can-i --list --output=yaml"
 os::cmd::expect_success "oc policy can-i --list --output=json"
-os::cmd::expect_success "oc policy can-i --list --output=wide"
```

Comment:
Why removing this test, isn't wide supported anymore?

Reply:
Yes, by removing the humanreadable printer, and moving the "normal" output to the success printer, we remove the need for this output format as well